features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h

```diff
@@ -23,18 +23,34 @@
  * \brief Struct ws_rpl_info Wi-SUN router RPL information.
  */
 typedef struct ws_rpl_info {
-    /** Address prefix given to devices in network  set to 0 if not available*/
-    uint8_t ipv6_prefix[8];
-    /** IID of router */
-    uint8_t ipv6_iid[8];
+    /** IPv6 Global Address of Router Node*/
+    uint8_t global_addr[16];
```

Comment:
These are different things and can't be changed Prefix might be the same or might come from different route